src/Storefront/Resources/views/storefront/page/account/order-history/order-detail.html.twig

```diff
@@ -200,6 +200,20 @@
                                 </div>
                             </div>
                         {% endblock %}
+
+                        {% block page_account_order_item_detail_table_footnote %}
+                            <div class="order-item-detail-footnote">
+                                {% if context.taxState == "gross" %}
```

Comment:
@JoshuaBehrens Shouldn't this be `order.taxStatus` instead? Right now it uses the context similar to the footer and would be the same for every order?

```suggestion
                                {% if order.taxStatus == "gross" %}
```

Reply:
Very good hint

Reply:
Nothing to do with your change but 😆 

![image](https://github.com/shopware/platform/assets/8480203/1fe5be9e-10ea-40d6-b571-f329d87f6674)